context.go

```diff
@@ -102,6 +107,26 @@ func CreateContext(masterKey, masterSalt []byte, profile ProtectionProfile, opts
 	return c, nil
 }
 
+// createContextWithCipher creates a new SRTP Context with a pre-created cipher. This is used for testing purposes only.
```

Comment:
If this is for tests only should it go in a `_test.go` file?

Reply:
Good idea, will move all test-related stuff to some _test.go file.